src/orchard/pp.clj

```diff
@@ -1,13 +1,14 @@
 (ns orchard.pp
   "A pretty-printer for Clojure data structures.
-
-  Based on the algorithm described in \"Pretty-Printing, Converting List
-  to Linear Structure\" by Ira Goldstein (Artificial Intelligence, Memo
-  No. 279 in Massachusetts Institute of Technology A.I. Laboratory,
-  February 1973)."
+  This namespace is borrowed from Eero Helenius' pp project and modified
```

Comment:
I think it's still a good idea to leave the reference to the underlying algorithm, as I doubt that everyone will go and check the upstream repo.

Reply:
(and perhaps high-level differences with the original implementation)